taskcluster/ac_taskgraph/__init__.py

```diff
@@ -23,7 +23,7 @@ def register(graph_config):
     Import all modules that are siblings of this one, triggering decorators in
     the process.
     """
-    _import_modules(["job", "routes", "target_tasks", "worker_types"])
+    _import_modules(["job", "routes", "target_tasks", "worker_types", "release-promotion"])
```

Comment:
I think you need an underscore here. `release_promotion`. This is probably what's causing the current decision task bustage, `[task 2020-12-04T00:56:51.079Z] ImportError: No module named release-promotion`

Reply:
oh right, my bad!